doc/tutorials/transformer/masked_modelling.ipynb

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
<p>Please, create a ticket, that this goes to the notebook connector.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>